src/app/admin/data-tables/page.tsx

```diff
@@ -1,26 +1,22 @@
 'use client';
-import tableDataDevelopment from 'variables/data-tables/tableDataDevelopment';
-import tableDataCheck from 'variables/data-tables/tableDataCheck';
-import CheckTable from 'components/admin/data-tables/CheckTable';
 import tableDataColumns from 'variables/data-tables/tableDataColumns';
-import tableDataComplex from 'variables/data-tables/tableDataComplex';
-import DevelopmentTable from 'components/admin/data-tables/DevelopmentTable';
 import ColumnsTable from 'components/admin/data-tables/ColumnsTable';
-import ComplexTable from 'components/admin/data-tables/ComplexTable';
+import { toast } from 'react-toastify';
+import useApi from 'app/hooks/useApi';
 
 const Tables = () => {
+  const api = useApi();
+  const onSubmit = async () => {
+    try {
+      const res = await api.get('user/all?page=1&limit=10');
```

Comment:
Do not hard code the parameters: page , limit

Reply:
its just a test if useApi run or not